changelog.md

```diff
@@ -3,6 +3,7 @@
 ## 0.4.3
 
 - Update `gproc` dependency to 0.9.0.
+- Add `format_status/1` for OTP 25
 - Add `format_status/1` for OTP 25.
```

Comment:
I applied your suggestion in an amended commit, so there are now two of these lines. 🥲 Should've mentioned before.

Reply:
Force-pushed again. Your suggestion incorporated in c8392aa45526d094eaa8823c7a0cbe79ba56a6e3.